openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,38 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            resources = {}
+            extra_values = {}
+            if 'load' in data['resources']:
+                extra_values['load'] = {
+                    'load_1': 100 * data['resources']['load'][0],
+                    'load_15': 100 * data['resources']['load'][2],
+                }
+                resources['load'] = 100 * data['resources']['load'][1]
+            if 'disk_usage' in data['resources']:
+                used_space, total_size = 0, 0
+                for disk in data['resources']['disk_usage']:
```

Comment:
Remember to change the key names: https://github.com/openwisp/lua-monitoring/pull/11/files#r425648187 and save **all** received raw data.

Also notice that you will be receiving `data['resources']['disk']['used_percent']` so you don't need to divide used_space by total_size. Also don't modify the value *100, just save the raw data at the moment.